reset.sh

```diff
@@ -0,0 +1,122 @@
+source .env
```

Comment:
useful :) but maybe something you might not want to upload in case someone accidentallly runs this and deletes everything?